dosagelib/plugins/d.py

```diff
@@ -27,6 +27,23 @@ class DangerouslyChloe(ComicControlScraper):
     firstStripUrl = url + 'strips-dc/Chapter_1_-_That_damned_girl'
 
 
+class DarkLegacy(ParserScraper):
+    # it is a bit more complicated due how the site is setup
+    import requests
+    import re
+    base_url = 'https://www.darklegacycomics.com'
+    response = requests.get(base_url + "/newest", allow_redirects=False)
```

Comment:
Please don't use requests directly, use the session attribute available to every module, for example like `MyLifeWithFel` here:

https://github.com/webcomics/dosage/blob/e2fb9a1c40be4dc61b502f5775cd5fb115906bbb/dosagelib/plugins/m.py#L243-L247

But since this is a common pattern, you probably want to use something like `indirectStarter` instead. Code is here:

https://github.com/webcomics/dosage/blob/e2fb9a1c40be4dc61b502f5775cd5fb115906bbb/dosagelib/helpers.py#L54-L64

There are many examples in other modules, like this one:

https://github.com/webcomics/dosage/blob/e2fb9a1c40be4dc61b502f5775cd5fb115906bbb/dosagelib/plugins/s.py#L80-L83

Reply:
Alright, gotcha. I'll fix those as you say